lib/facter/lspci.rb

```diff
@@ -0,0 +1,65 @@
+#
+# Fact: lspci
+#
+# Purpose: get the results of lspci
+#
+# Resolution:
+#   Uses lspci
+#
+# Caveats:
+#   Needs lspci in path
+#
+# Notes:
+#   The result is [class][vendor][slot][attr] = value
+#
+if Facter::Util::Resolution.which('lspci')
```

Comment:
This is will end up running on any system for which `lspci` resolves.  It would be better for this code block to only execute if the `confine` on the fact is met.